src/libs/AxelarQueryAPI.ts

```diff
@@ -108,7 +110,12 @@ interface HopParams {
    * Token amount (in units) that is used in callContractWithToken for checking if express is supported
    */
   amountInUnits?: string;
-}
+};
+
+/**
+ * Parameters for estimating the fee for an ITS transfer
+ */
+type ItsFeeParams = BaseFeeParams;
```

Comment:
should avoid reusing the params and create one that makes sense for ITS specifically